analyzers/its/sources/SonarLintExclusions/global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "7.0.100-rc.1.22431.12",
```

Comment:
We can target the released version of .NET7 no need to use the release candidate.